src/System.Console/pkg/System.Console.pkgproj

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Version>4.0.1</Version>
+  </PropertyGroup>
```

Comment:
nit: You can avoid duplicating this across the unix/win pkgproj files if you add a new dir.props to the system.console/pkg directory a la https://github.com/dotnet/corefx/pull/28111/files#diff-81e8a10ad879938700c91a3ce2935132R6

Reply:
Will do.